### ISSUE TITLE
google-api-client-protobuf

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/google-api-java-client#783]
```
External references, such as a standards document, or specification?

http://javadoc.google-http-java-client.googlecode.com/hg/1.15.0-rc/com/google/ap
i/client/protobuf/package-frame.html

http://javadoc.google-http-java-client.googlecode.com/hg/1.15.0-rc/com/google/ap
i/client/http/protobuf/package-frame.html

http://javadoc.google-api-java-client.googlecode.com/hg/1.15.0-rc/com/google/api
/client/googleapis/json/package-summary.html

http://javadoc.google-api-java-client.googlecode.com/hg/1.15.0-rc/com/google/api
/client/googleapis/batch/json/package-summary.html

http://javadoc.google-api-java-client.googlecode.com/hg/1.15.0-rc/com/google/api
/client/googleapis/services/json/package-summary.html

http://javadoc.google-api-java-client.googlecode.com/hg/1.15.0-rc/com/google/api
/client/googleapis/subscriptions/json/package-summary.html

http://javadoc.google-api-java-client.googlecode.com/hg/1.15.0-rc/com/google/api
/client/googleapis/testing/services/json/package-summary.html

Java environments (e.g. Java 6, Android 2.3, App Engine, or All)?

All

Please describe the feature requested.

Basically everything we have for JSON we also need for protobuf.  We should 
create a new artifact google-api-client-protobuf that depends on 
google-http-client-protobuf.
```

Original issue reported on code.google.com by `yan...@google.com` on 18 May 2013 at 11:21

**Comment by GoogleCodeExporter** [comment]
Original comment by `yan...@google.com` on 26 Jun 2013 at 7:04
- Changed state: **Fixed**
- Added labels: **Milestone-Version1.16.0**, **Priority-Medium**
- Removed labels: **Milestone-Version1.17.0**
